tests/FSharpPlusFable.Tests/FSharpTests/Extensions.fs

```diff
@@ -87,4 +93,11 @@ let ExtensionsTest =
                    equal (mapItem2 string (1,2,3)) (1,"2",3)
                    equal (item3 (1,2,3)) 3
                    )
+
+      testCase "eq on DList 1" (fun () -> equal true  (dlistA = dlistB))
+      testCase "eq on DList 2" (fun () -> equal false (dlistA = dlistC))
+      testCase "eq on DList 3" (fun () -> equal true  ((dlistA :> obj) = (dlistB :> obj)))
+      testCase "eq on DList 4" (fun () -> equal false ((dlistA :> obj) = (dlistC :> obj)))
+      testCase "eq on DList 5" (fun () -> equal true  ((dlistA :> obj) = (dlistD :> obj))) // this behavior differs from (non-fable) F# but same way it would be with normal lists.
```

Comment:
Note that the last test differs in Fable.
But that's also true for normal lists, I think there's no way around it since JS doesn't carry type information.

Reply:
Unless you add additional encoding using something like ```{"$type":"obj", "value":1}``` in order to include the additional information at runtime.